lib/src/searchfield.dart

```diff
@@ -752,6 +751,7 @@ class _SearchFieldState<T> extends State<SearchField<T>> {
       filteredResult.addAll(widget.suggestions);
       // hide the suggestions
       suggestionStream.sink.add(null);
+      searchController!.text = item.searchKey;
```

Comment:
Good catch on this one.

Reply:
If I remove this line, then this line fails

expect((textField.evaluate().first.widget as TextFormField).controller?.text, 'DEF');

I will investigate why is that

Reply:
Yes your change is valid. That was my point before.

Reply:
so, ok to merge now?

Reply:
@maheshj01  please check it now